src/util/dstring.h

```diff
@@ -73,6 +73,16 @@ class dstringt final
 
   // access
 
+  // In some situations we need to check for if this string has been
+  // correctly shared before doing an operation on it.
+
+  // Don't remove, used in python pretty-printers. (Also clang is wrong
+  // about it always being false)
+  bool uninitialized() const
```

Comment:
Can we actually make this `private`? I don't think gdb will enforce access rules?

Reply:
Can't due to the way python/gdb interacts (or at least the way it's being used here), but I can put DEPRECATED on it to make sure it's not called in the C++.

Reply:
Ah, don't worry too much, there seems to be no risk in actually calling it, it would just be useless :-)